docs/Other SDL Features/Setting the Built-in Nav System Destination/index.md

```diff
@@ -45,10 +48,23 @@ sdlManager.start { (success, error) in
     }
 }
 ```
+!@
+
+@![android, javaSE, javaEE]
+```java
+HMICapabilities hmiCapabilities = (HMICapabilities) sdlManager.getSystemCapabilityManager().getCapability(SystemCapabilityType.HMI);
+if (hmiCapabilities.isNavigationAvailable()) {
```

Comment:
I don't think this is how this property works. It only tells us if there's built in nav. Ideally you should use the `navigationCapability` property to see if `sendLocation` is enabled. However, that property is newer than `sendLocation` itself.

Reply:
Fixed. Added more info on checking permissions